langwatch/src/optimization_studio/utils/dslUtils.ts

```
@@ -0,0 +1,46 @@
+import type { Node } from "@xyflow/react";
+
+export const clearDsl = (
+  dsl: z.infer<typeof workflowJsonSchema>,
+  includeExecutionStates = false
+) => {
+  return {
+    ...dsl,
+    version: undefined,
+    edges: dsl.edges.map((edge) => {
+      const edge_ = { ...edge };
+      delete edge_.selected;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)